src/views/AddItem.jsx

```diff
@@ -20,8 +21,23 @@ export function AddItem({ listToken }) {
 		setItem({ ...item, [e.target.name]: updateVal });
 	};
 
+	const isInvalid = (name) => {
+		if (isEmpty(name)) {
+			setStatus('Can not add an empty item');
+			setItem(defaultItem);
```

Comment:
This is a nice touch for UX although I wonder - since the user can't see what they typed before, might they want the previous input to linger for a sec before it clears out? In case of typo of a genuinely new item. Currently, they would probably go to List page and check with the filter?

Reply:
I agree!

We should try to figure out a way to make the last input visible, either by waiting to reset the text-input, or maybe adding the item name to the error status message. For example: `"duplicate item" already added to list`

Reply:
Adjusted this ! 👩🏼‍🍳 💋  (my heart wished for a **chef's kiss** emoji as I have on in Slack but this must do. Imagine)